watertap/unit_models/anaerobic_digestor.py

```diff
@@ -485,7 +485,7 @@ def build(self):
         )
         # The value is taken from Maravelias' data
         self.energy_electric_flow_vol_inlet = Param(
-            initialize=59.99279,
+            initialize=3.35,
```

Comment:
Wow, still fairly substantial considering that seawater RO desalination is somewhere around this value. I wouldn't have expected that. This may be correct based on values in the cited literature, but we might want to reinspect in the near-term to make sure it makes sense to us. That said, it shouldn't hold this PR.